src/driver/amdxdna/aie2_smu.c

```diff
@@ -268,71 +162,9 @@ void aie2_smu_stop(struct amdxdna_dev_hdl *ndev)
 	int ret;
 
 	/* Minimize clocks/dpm level prior to power off */
-	ret = aie2_smu_set_dpm_level(ndev, 0);
-	if (ret)
-		XDNA_WARN(ndev->xdna, "Set dpm level 0 failed, ret %d", ret);
+	ndev->priv->hw_ops.set_dpm(ndev, 0);
 
 	ret = aie2_smu_set_power_off(ndev);
 	if (ret)
 		XDNA_WARN(ndev->xdna, "Power off failed, ret %d", ret);
 }
-
-static void aie2_smu_npu1_hack(struct amdxdna_dev_hdl *ndev)
```

Comment:
We are getting rid of  aie2_smu_npu1_hack that differentiates between the dpm_levels of npu1 variants.
 where are we distinguishing that?
 
 Can this change be verified both NPU1 variants?

Reply:
I chatted with Lizhi and clarified the flow.
I also verified on the NPU1 variant2 and it works as expected.